octavia_f5/restclient/as3objects/application.py

```diff
@@ -19,5 +19,4 @@
 
 
 def get_name(loadbalancer_id):
-    return constants.PREFIX_LOADBALANCER + loadbalancer_id.\
-        replace('/', '').replace('-', '_')
```

Comment:
The changenotes don't seem to say that slashes (`/`) are now allowed.

Reply:
They are no slashes to escape in UUIDs: https://de.wikipedia.org/wiki/Universally_Unique_Identifier

Reply:
Makes sense. I guess I was just wondering why slashes were being removed in the first place.